packages/web/src/lib/posthogEvents.ts

```
@@ -250,6 +250,18 @@ export type PosthogEventMap = {
     $pageview: {
         $current_url: string,
     },
+    //////////////////////////////////////////////////////////////////
+    wa_requests_list_approve_request_success: {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)